### PR TITLE
Update reference to check definition schema

### PR DESCRIPTION
**Pull request by nelsonkopliku**
Uses latest checks definition schema.

Allows wanda and premium checks pipelines to pass with the latest changes to metadata.

### CODE REVIEW

File: src/dsl/validation.rs [hunk 1]
```diff
@@ -461,15 +466,15 @@ mod tests {
...
-        let deserialization_result = serde_yaml::from_str::<Check>(&input);
+        let deserialization_result = serde_yaml::from_str::<Check>(input);
 
         let json_schema = get_json_schema();
         let validation_result = validate(&json_value, "156F64", &json_schema, &engine);
 
-        assert_eq!(validation_result.is_ok(), true);
-        assert_eq!(deserialization_result.is_ok(), true);
+        assert!(validation_result.is_ok());
+        assert!(deserialization_result.is_ok());
```
Comment (dottorblaster) [inline src/dsl/validation.rs:477]: Sorry, Github ate my comment. Would you add metadata to this test or write another one with metadata and use this set of assertions? I'd like to see if also the deserialization into `Check` is fine as well

Comment (nelsonkopliku) [reply src/dsl/validation.rs:480]: I ended up in adding metadata to the `Check` struct and adding the deserialization result assertions in the `validate_ok_check` test

**Review by dottorblaster — APPROVED** [review]
LGTM

File: src/dsl/validation.rs (hunk 1 above)
Comment (dottorblaster) [reply src/dsl/validation.rs:480]: Perfect, thanks :+1:

**Pull request merged**